client/assets/practice.js

```diff
@@ -0,0 +1,62 @@
+function initialise() {
+    let form = document.getElementById('sbox')
+    form.addEventListener('submit', handleFormSubmit)
+}
+
+function handleFormSubmit(e){
+    e.preventDefault()
+    let searchTerm = e.target.sboxspace.value;
+    let route = e.submitter.id;
```

Comment:
We like that you've used the submitter ids to determine which button was clicked, meaning that you only had to have one event listener!

Reply:
Can you expand on this? As far as I can see theres only one event listener to track the entire form